CHANGES.md

```diff
@@ -1,3 +1,10 @@
+0.17.4 (2018-03-20)
```

Comment:
Hi! 

Thanks for adding a section in CHANGES!

The release number and its date is not the same activity as adding items to this list, so perhaps the heading could be _Unreleased_ instead of _0.17.4 (2018-03-20)_?

Reply:
I fixed this in 3b05356